src/kernels/composable_kernel/include/tensor_operation/gridwise_gemm_xdlops.hpp

```diff
@@ -325,7 +318,7 @@ struct GridwiseGemmTransposedANormalBNormalCXdlops_v1
                      m_thread_data_on_global % (M2 * M1) / M2,
                      m_thread_data_on_global % M2,
                      n_thread_data_on_global})
-                    .Run(p_c_thread + i * BlkSize, p_c_global);
+                    .Run(c_thread_vec.n + i * BlkSize, p_c_global);
```

Comment:
Do you need a #pragma unroll above ? Line 294

not comment, just question:
Is this the only place in the code where pointer ```c_thread_vec.n``` is used to access c_thread_vec?

Reply:
Yes. #pragma unroll is required to eliminate stack.

Yes. c_thread_vec.n is only used here.